### PR TITLE
Update Horizon 0.24.0 CHANGELOG

### PR DESCRIPTION
**Pull request by bartekn**

### CODE REVIEW

File: services/horizon/CHANGELOG.md [hunk 1]
```diff
@@ -8,8 +8,21 @@ bumps.  A breaking change will get clearly notified in this log.
...
+* Provide mechanism for retrying XDR stream errors ([#1899](https://github.com/stellar/go/pull/1899)).
+* Emit errors level log after 3 failed attempts to validate state ([#1918](https://github.com/stellar/go/pull/1918)).
+* Fixed out of bounds error in ledger backend reader ((#1914)[https://github.com/stellar/go/pull/1914]).
+* Fixed out of bounds error in URL params handler ((#1973)[https://github.com/stellar/go/pull/1973]).
+* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1950)).
+* Fixed multiple issues connected to graceful shutdown of Horizon.
+
+### Scheduled Breaking Changes
+
+* All `*_accepted_fee` fields in `/fee_stats` endpoint are deprecated. Fields will be removed in Horizon 0.25.0.
```
Comment (bartekn) [inline services/horizon/CHANGELOG.md]: Do you think we should remind about other breaking changes in 0.25.0?

Comment (abuiles) [reply services/horizon/CHANGELOG.md]: @bartekn sounds like a good idea!

File: services/horizon/CHANGELOG.md [hunk 2]
```diff
@@ -8,8 +8,21 @@ bumps.  A breaking change will get clearly notified in this log.
...
 ## v0.24.0
 
-* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1950))
-* Improved performance of asset stats processor ([#1987](https://github.com/stellar/go/pull/1987))
+* Add `fee_charged` and `max_fee` objects to `/fee_stats` endpoint ([#1964](https://github.com/stellar/go/pull/1964)).
+* Experimental ledger ingestion processor ([#1949](https://github.com/stellar/go/pull/1949)).
+* Improved performance of asset stats processor ([#1987](https://github.com/stellar/go/pull/1987)).
+* Provide mechanism for retrying XDR stream errors ([#1899](https://github.com/stellar/go/pull/1899)).
+* Emit errors level log after 3 failed attempts to validate state ([#1918](https://github.com/stellar/go/pull/1918)).
+* Fixed out of bounds error in ledger backend reader ((#1914)[https://github.com/stellar/go/pull/1914]).
```
Comment (bartekn) [inline services/horizon/CHANGELOG.md]:
```suggestion
* Fixed out of bounds error in ledger backend reader ([#1914](https://github.com/stellar/go/pull/1914)).
```

File: services/horizon/CHANGELOG.md [hunk 3]
```diff
@@ -8,8 +8,21 @@ bumps.  A breaking change will get clearly notified in this log.
...
+* Fixed out of bounds error in URL params handler ((#1973)[https://github.com/stellar/go/pull/1973]).
```
Comment (bartekn) [inline services/horizon/CHANGELOG.md]:
```suggestion
* Fixed out of bounds error in URL params handler ([#1973](https://github.com/stellar/go/pull/1973)).
```

File: services/horizon/CHANGELOG.md [hunk 4]
```diff
@@ -8,8 +8,21 @@ bumps.  A breaking change will get clearly notified in this log.
 
 ## v0.24.0
 
-* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1950))
-* Improved performance of asset stats processor ([#1987](https://github.com/stellar/go/pull/1987))
+* Add `fee_charged` and `max_fee` objects to `/fee_stats` endpoint ([#1964](https://github.com/stellar/go/pull/1964)).
+* Experimental ledger ingestion processor ([#1949](https://github.com/stellar/go/pull/1949)).
+* Improved performance of asset stats processor ([#1987](https://github.com/stellar/go/pull/1987)).
+* Provide mechanism for retrying XDR stream errors ([#1899](https://github.com/stellar/go/pull/1899)).
+* Emit errors level log after 3 failed attempts to validate state ([#1918](https://github.com/stellar/go/pull/1918)).
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md]:
```suggestion
* Emit error level log after 3 failed attempts to validate state ([#1918](https://github.com/stellar/go/pull/1918)).
```

File: services/horizon/CHANGELOG.md [hunk 5]
```diff
@@ -8,8 +8,21 @@ bumps.  A breaking change will get clearly notified in this log.
...
+* Fixed out of bounds error in ledger backend reader ([#1914](https://github.com/stellar/go/pull/1914)).
+* Fixed out of bounds error in URL params handler ([#1973](https://github.com/stellar/go/pull/1973)).
+* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1950)).
+* Fixed multiple issues connected to graceful shutdown of Horizon.
+
+### Scheduled Breaking Changes
+
+* All `*_accepted_fee` fields in `/fee_stats` endpoint are deprecated. Fields will be removed in Horizon 0.25.0.
+
+Check [Beta Testing New Ingestion System](https://github.com/stellar/go/blob/master/services/horizon/internal/expingest/BETA_TESTING.md) if you want to test new ingestion system.
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md]:
```suggestion
Check [Beta Testing New Ingestion System](https://github.com/stellar/go/blob/master/services/horizon/internal/expingest/BETA_TESTING.md) if you want to test the new ingestion system.
```

File: services/horizon/CHANGELOG.md [hunk 6]
```diff
@@ -8,8 +8,21 @@ bumps.  A breaking change will get clearly notified in this log.
...
-* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1950))
-* Improved performance of asset stats processor ([#1987](https://github.com/stellar/go/pull/1987))
+* Add `fee_charged` and `max_fee` objects to `/fee_stats` endpoint ([#1964](https://github.com/stellar/go/pull/1964)).
+* Experimental ledger ingestion processor ([#1949](https://github.com/stellar/go/pull/1949)).
+* Improved performance of asset stats processor ([#1987](https://github.com/stellar/go/pull/1987)).
+* Provide mechanism for retrying XDR stream errors ([#1899](https://github.com/stellar/go/pull/1899)).
+* Emit errors level log after 3 failed attempts to validate state ([#1918](https://github.com/stellar/go/pull/1918)).
+* Fixed out of bounds error in ledger backend reader ([#1914](https://github.com/stellar/go/pull/1914)).
+* Fixed out of bounds error in URL params handler ([#1973](https://github.com/stellar/go/pull/1973)).
+* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1950)).
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md]:
```suggestion
* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1952)).
```

File: services/horizon/CHANGELOG.md [hunk 7]
```diff
@@ -8,8 +8,21 @@ bumps.  A breaking change will get clearly notified in this log.
 
 ## v0.24.0
 
-* Rename `OperationFeeStats` to `FeeStats` ([#1950](https://github.com/stellar/go/pull/1950))
-* Improved performance of asset stats processor ([#1987](https://github.com/stellar/go/pull/1987))
+* Add `fee_charged` and `max_fee` objects to `/fee_stats` endpoint ([#1964](https://github.com/stellar/go/pull/1964)).
+* Experimental ledger ingestion processor ([#1949](https://github.com/stellar/go/pull/1949)).
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md]:
```suggestion
* Experimental ledger header ingestion processor ([#1949](https://github.com/stellar/go/pull/1949)).
```

File: services/horizon/CHANGELOG.md (hunk 1 above)
Comment (ire-and-curses) [reply services/horizon/CHANGELOG.md]: yeah I agree. I can add that in a separate PR if you like.

**Comment by bartekn** [comment]
@ire-and-curses added. If I'm missing something please update this PR and merge.

**Pull request merged**